tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/FindReplaceUITest.java

```diff
@@ -263,9 +249,6 @@ public void testReplaceAllInSelection() {
 		fTextViewer.setSelection(new TextSelection(4 + System.lineSeparator().length(), 8 + System.lineSeparator().length()));
 		initializeFindReplaceUIForTextViewer();
 
-		dialog.unselect(SearchOptions.GLOBAL);
-		dialog.unselect(SearchOptions.WHOLE_WORD);
```

Comment:
Why do you remove this?

Reply:
The FindReplaceOverlayTest still works correctly with those lines in. I will add them back